.github/workflows/ci.yml

```diff
@@ -45,7 +45,9 @@ jobs:
         with:
           submodules: true
       - run: echo "LIBCLANG_PATH=$((gcm clang).source -replace "clang.exe")" >> $env:GITHUB_ENV
-      - uses: dtolnay/rust-toolchain@stable
+      - uses: dtolnay/rust-toolchain@nightly
```

Comment:
As I said above, I don't think we should build the workspace on nightly. Everything except for the examples should be built with stable rust.

Reply:
Ok let me process with the changes, I'll try to be quick.

Reply:
Sorry I didn't understand correctly at first. Let's see the pipeline, it should be fine by now.

Reply:
@jac3km4 should we have CI test run also on nightly ? or change the test command to not include `--all-features` (which is the reason why `cargo test` is also run for `example`) ?